combine/usergroup.go

```diff
@@ -0,0 +1,53 @@
+package combine
+
+import (
+	"io"
+	"os"
+	"strconv"
+)
+
+const userGroupSumCols = 3
+const intBase = 10
+
+func MergeUserGroupFiles(inputs []*os.File, output *os.File) error {
+	return MergeAndCompress(inputs, output, mergeUserGroupStreamToCompressedFile)
+}
+
+// mergeUserGroupStreamToCompressedFile merges pre-sorted (pre-merged) usergroup
+// data (eg. from a `sort -m` of .byusergroup files), summing consecutive lines
+// with the first 3 columns, and outputting the results to a file, compressed.
+func mergeUserGroupStreamToCompressedFile(data io.ReadCloser, output io.Writer) error {
```

Comment:
No need of this method since it doesn't do anything the MergeSummaryLines call inside does. Just do that directly.

Reply:
Not deleted yet. Just call MergeSummaryLines inside UserGroupFiles().